### PR TITLE
Fix JSON formatting

### PR DESCRIPTION
**Pull request by ry**
Since https://github.com/denoland/deno/commit/d9b1f96897239bf7de8cdfd11d40e3f99bb29a4a landed in CLI, CI is broken here.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ jobs:
         run: deno upgrade --canary
 
       - name: Format
-        run: deno fmt --check
+        run: deno fmt --check --ignore=fs/testdata/json_invalid.json
```
Comment (bartlomieju) [inline .github/workflows/ci.yml:63]: Again, this should be removed and inline ignore should be added to the file

Comment (ry) [reply .github/workflows/ci.yml:63]: How does that work?

Comment (bartlomieju) [reply .github/workflows/ci.yml:63]: Add `// deno-fmt-ignore-file` in the appropriate file

Comment (ry) [reply .github/workflows/ci.yml:63]: but it's json..

Comment (bartlomieju) [reply .github/workflows/ci.yml:63]: Right... This is unlucky, we could fix that by changing it to JSONC

**Comment by ry** [comment]
-> #744

**Pull request closed without merging**